src/domain_bridge/domain_bridge.cpp

```diff
@@ -145,25 +170,47 @@ class DomainBridgeImpl
       return;
     }
 
+    // Create a null entry to avoid duplicate bridges
+    bridged_topics_[topic_bridge] = {nullptr, nullptr};
```

Comment:
The condition at L166 above should be guarding against overriding existing entries. The reason for adding an entry here, and not in the callback, is to avoid multiple callbacks from adding the same entry.